File: idea/Aplicacion/src/Logica/ContactosPendientes.java
package Logica;

import java.util.Scanner;

class ContactosPendientes extends Contacto{
	private final String targeta;

	protected ContactosPendientes(String nombre, String email, String nombreCompleto) {
		super(nombre, email, nombreCompleto);
		this.targeta = "No hay mensajes";
	}

	protected String getTargeta() {
		return targeta;
	}
	
	protected static ContactosPendientes crearContactoPendiente(Scanner sc, Scanner scl) {
		System.out.println("Crear Contacto");
		System.out.print("Nombre: ");
		String nombre = scl.nextLine();
		System.out.print("Email: ");
		String email = sc.next();
		System.out.print("Nombre completo: ");
		String nombreCompleto = sc.nextLine();
		return new ContactosPendientes(nombre, email, nombreCompleto);
	}
}

File: Python/gestorAplicacion/Hija/contactosPendientes.py
#Los contactos pendientes son personas con las que no se a iniciado un conversacion.

class ContactosPendientes(Contacto):

        
        
    #Crea el contacto sin mensajes
    def __init__(self, nombre, email, nombreCompleto):
        super().__init__(nombre, email, nombreCompleto)
        self._mensajes = "No hay mensajes"

    def getMensajes(self):
        return self._mensajes

    def setMensajes(self, mensajes):
        self._mensajes = mensajes

    def __str__(self):
        return super.__str__()

    def getNombre(self):
        return super.getNombre()
    
    def setNombre(self, nombre):
        super.setNombre(nombre)

    def getEmail(self):
        return super.getEmail()
    
    def setEmail(self, email):
        super.setEmail(email)

    def getNombreCompleto(self):
        return super.getNombreCompleto()
    
    def setNombreCompleto(self, nombreCompleto):
        super.setNombreCompleto(nombreCompleto)
